### PR TITLE
Enable add usings on paste by default

### PR DESCRIPTION
**Pull request by ryzngard**
With the new experience no longer blocking the UI thread, we want to enable this feature by default in 17.3

### CODE REVIEW

File: src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs [hunk 1]
```diff
@@ -94,9 +94,9 @@ public FeatureOnOffOptions()
             FeatureName, "NavigateAsynchronously", defaultValue: true,
             storageLocation: new RoamingProfileStorageLocation("TextEditor.NavigateAsynchronously"));
 
-        public static readonly PerLanguageOption2<bool?> AddImportsOnPaste = new(
-            FeatureName, "AddImportsOnPaste", defaultValue: null,
-            storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste"));
+        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new(
+            FeatureName, "AddImportsOnPaste", defaultValue: true,
+            storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2"));
```
Comment (davidwengier) [inline src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs:99]: Perhaps a comment indicating to future explorers why the "2" suffix is here

**Review by davidwengier — APPROVED** [review]

**Review by dibarbet — APPROVED** [review]
is the plan to entirely remove the option in 17.4?

**Comment by ryzngard** [comment]
> is the plan to entirely remove the option in 17.4?

Maybe... We'll have to discuss that. I could still see this behavior being undesirable to some people. We should bring it up in a design meeting though and see what the team thinks.

**Comment by CyrusNajmabadi** [comment]
> s the plan to entirely remove the option in 17.4?

Good question.  I could see people never wanting this feature and potentially disabling it.  I've even noticed it a few times when it popped up and i didn't need it.  It didn't hugely bother me, but it might for other people.

**Comment by ryzngard** [comment]
>  I've even noticed it a few times when it popped up and i didn't need it

Currently working on a PR to help here. 

> It didn't hugely bother me, but it might for other people.

I'm always hesitant to make things like this not configurable since it is visual noise and potentially modifying code for a user in a way they can't readily see. It would be similar to having to undo IntelliCode suggestions every time or something.... There's a lot of impact here for various user groups that could be VERY negative.

**Comment by CyrusNajmabadi** [comment]
> Currently working on a PR to help here.

Oh?  What are you thinking about?

**Comment by sharwell** [comment]
> is the plan to entirely remove the option in 17.4?

As long as the feature is only an approximation of correct (which it is until it uses the original source location to obtain symbol information for semantically-correct binding on paste), I don't think we can remove the option.

**Comment by dibarbet** [comment]
> As long as the feature is only an approximation of correct (which it is until it uses the original source location to obtain symbol information for semantically-correct binding on paste), I don't think we can remove the option.

this is a fair point - I hit this at least once since I wrote the original comment.

**Pull request merged**